tests/framework/utils/k8s_helper.go

```diff
@@ -395,4 +410,19 @@ func (k8sh *K8sHelper) WaitUntilPodIsDeleted(podNamePattern string) bool{
 		time.Sleep(3 * time.Second)
 	}
 	return false
-}
\ No newline at end of file
+}
+
+func (k8sh *K8sHelper) WaitUntilPVCIsBound(pvcname string) bool {
+
+	inc := 0
+	for inc < 30 {
+		out, _ := k8sh.GetPVCStatus(pvcname)
```

Comment:
is there any error state we should watch for? Otherwise in an error condition we will have to wait for timeout after the full retry loop